doc/jemalloc.xml.in

```diff
@@ -2623,8 +2629,8 @@ struct extent_hooks_s {
           <literal>r-</literal>
           [<option>--enable-stats</option>]
         </term>
-        <listitem><para>Cumulative number of small allocation requests.
-        </para></listitem>
+        <listitem><para>Cumulative number of allocation requests satisfied by
+        all bin size classes.</para></listitem>
```

Comment:
The addition in ```nmalloc``` does help.

I just realized, this should be "all small bin size".